subsys/bluetooth/mesh/Kconfig

```diff
@@ -660,7 +660,15 @@ config BT_MESH_MODEL_EXTENSIONS
 	  Enable support for the model extension concept, allowing the Access
 	  layer to know about Mesh model relationships.
 
-if BT_SETTINGS
+menuconfig BT_MESH_CONTINUOUS_STORE
```

Comment:
Wouldn't `AUTOMATIC_STORE` be more descriptive, since if you don't set this it means you have to manually request storing?